WorldBank-Transport/DRIVER
# Use auto-incrementing labels for related items

## Overview

Users were confused about the way we were handling labels of related objects. We decided to instead opt for using the object name followed by an auto-incrementing label. Some of the required changes for this were done in the `json-editor` -- see https://github.com/azavea/json-editor/pull/11

### Checklist


### Demo

![Screenshot_2019-04-18 DRIVER](https://user-images.githubusercontent.com/6386/56389025-8869e080-61f6-11e9-9224-58d7e3163e41.png)

![Screenshot_2019-04-18 DRIVER(2)](https://user-images.githubusercontent.com/6386/56389134-c23ae700-61f6-11e9-804b-949603270be2.png)

### Notes

This intentionally doesn't alter any behavior of the Android app, and will also not interfere with how the Android app is performing the labeling. There may be future changes to the Android app to align it appropriately.

## Testing Instructions

 * Provision your `app` VM and load the site up in a browser
 * Go to the record list view and experiment with viewing/creating/updating records with relationships
 * Verify the auto-incremented labels are displayed in each place the relationships are displayed or are selectable

Closes [PT164899426](https://www.pivotaltracker.com/story/show/164899426)
